lms/migrations/versions/79eda94de79f_add_organizations_model.py

```diff
@@ -0,0 +1,69 @@
+"""
+Add Organizations model.
+
+Revision ID: 79eda94de79f
+Revises: 396f46318023
+Create Date: 2022-09-07 16:57:14.077132
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "79eda94de79f"
+down_revision = "396f46318023"
```

Comment:
There's a new migration in main so this needs to be rebased and this value changed to `9bb2beba95bc`